ddtrace/contrib/pytest/plugin.py

```diff
@@ -123,8 +123,14 @@ def pytest_runtest_protocol(item, nextitem):
         # Parameterized test cases will have a `callspec` attribute attached to the pytest Item object.
         # Pytest docs: https://docs.pytest.org/en/6.2.x/reference.html#pytest.Function
         if getattr(item, "callspec", None):
-            params = {"arguments": item.callspec.params, "metadata": {}}
-            span.set_tag(test.PARAMETERS, json.dumps(params, default=repr))
+            parameters = {"arguments": {}, "metadata": {}}
```

Comment:
can we add some type hinting on this section of code?

since this should be:

```
parameters: dict[str, dict[str, str]] = {}
```

right?